descriptor.md

```diff
@@ -0,0 +1,123 @@
+<!--[metadata]>
++++
+draft = true
++++
+<![end-metadata]-->
+
+# OpenContainers Content Descriptors
+
+OCI have several components that come to together to describe an image.
+References between components form a [Merkle Directed Acyclic Graph (DAG)](https://en.wikipedia.org/wiki/Merkle_tree).
+The references in the _Merkle DAG_ are expressed through _Content Descriptors_.
+A _Content Descriptor_ or _Descriptor_, describes the disposition of targeted content.
```

Comment:
On Fri, Jun 03, 2016 at 11:38:09AM -0700, Stephen Day wrote:
> > +The references in the _Merkle DAG_ are expressed through _Content Descriptors_.
> > +A _Content Descriptor_ or _Descriptor_, describes the disposition of targeted content.
> 
> "disposition" precisely means "the way in which something is placed
> or arranged, especially in relation to other things".
> 
> However, "describes the disposition" stutters, as well. I'll tweak
> this. Any suggestions?

I'd just drop the line.  The previous line and rest of the docs make
the idea clear enough.